src/components/AddPeopleModal/AddPeopleModal.tsx

```diff
@@ -0,0 +1,166 @@
+import { useState } from 'react';
+import { Modal, Box, Typography, TextField, Button } from '@mui/material';
+
+type AddPeopleModalProps = {
```

Comment:
Suggestion to rename all People to Volunteer. We call it volunteer everywhere. And it is just volunteers.